example/integration_test/configuration_test.dart

```diff
@@ -134,4 +136,62 @@ void main() {
             (events[0]['event']['platform'] == 'iot')),
         isTrue);
   });
+
+  testWidgets("screenContext and applicationContext on by default",
+      (WidgetTester tester) async {
+    SnowplowTracker tracker = await Snowplow.createTracker(
+        namespace: 'screen-on', endpoint: SnowplowTests.microEndpoint);
+
+    await tracker
+        .track(const Structured(category: 'category', action: 'action'));
+
+    await SnowplowTests.checkMicroGood((dynamic events) {
+      if (events.length != 1) {
+        return false;
+      }
+      String eventType =
+          events[0]['event']['v_tracker'].toString().substring(0, 2);
+      // screenContext/applicationContext are not available on web
+      if (eventType == 'and' || eventType == 'ios') {
```

Comment:
Nit: I think it'd be a bit nicer to check for the platform at the beginning of the test and return early on Web with something like `if (!Platform.isAndroid && !Platform.isIOS) { return; }`, [see here](https://stackoverflow.com/a/50744481).